.gitignore

```diff
@@ -15,4 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-.idea
```

Comment:
Thanks for the tip - I have about 20 WebStorm projects and just keep repeating that =)